scripts/vault-init.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sleep 10
+# shellcheck source=/dev/null
+source $1/.env
+export VAULT_ADDR=http://127.0.0.1:$VAULT_GUEST_PORT
+docker exec -i vault sh -c "vault operator init -key-shares=5 -key-threshold=3 --address=$VAULT_ADDR" > keys.txt
```

Comment:
it works, but I am less fond of the `docker exec` method, it assumes there is a shell and it won't always be the case with all sort of images

Reply:
What is the way you do fond to accomplish this task?

Reply:
To the vault itself, add: `--name vault`
i.e:
`docker run --rm -e VAULT_ADDT=http://vault:8200 --network devopsloft_network ${NAMESPACE}/vault operator init ....`
Possibly wrap it in a function.
Will be easier with compose later.
Again, the `docker exec` works, the change is not a "must"

Reply:
As it is working for this image, I would suggest not to do the change you advised.